apps/aeutils/priv/aeternity_config_schema.json

```diff
@@ -1099,6 +1099,13 @@
                     "maximum" : 32,
                     "default" : 32
                 },
+                "ping_connected_peer_percentage" : {
```

Comment:
I'd go with just `ping_peers_percentage` name :)

Reply:
I'm adding a subgroup for pings, so I'll change the parameter to peer_percentage.